docs/deployment-patterns/multi-tenant-deployments/multi-tenant-deployment-guide/designing-a-multi-tenant-upgrade-process.md

```diff
@@ -4,201 +4,203 @@ description: How to design a multi-tenant upgrade process.
-![](images/5865746.png "width=500")
+![](images/multi-tenant-lifecycle.png "width=500")
 
-### Step 2: Configure a test tenant
+## Step 2: Configure a test tenant
 
 Now we will create a new tenant for one of the people in our quality assurance or test team.
 
 1. Configure a new tenant giving it any name and tags that make sense.
-2. Connect the tenant to the project and the **MT Test** environment.
+2. Connect the tenant to the project and the **MT Test** environment.
 
-### Step 3: Configure another tenant with multiple environments
+## Step 3: Configure another tenant with multiple environments
 
-Now we will configure a tenant to deploy into multiple environments. If you have been working through this guide you will already have a tenant you can use.
+Now we will configure a tenant to deploy into multiple environments. If you have been working through this guide, you will already have a tenant you can use.
 
-1. Connect a tenant to the project and both the **MT Staging** and **MT Production** environments.
+1. Connect a tenant to the project and the **MT Staging** and **MT Production** environments.
 
-### Step 4: Deploy
+## Step 4: Deploy
 
-## Deploying to Individual Tenants
+### Deploying to Individual Tenants
 
-You can deploy to a single tenant, or a handful of tenants in a batch by selecting them manually. This can be a good way to get started with multi-tenant deployments, and it's also a good fallback when you need it.
+You can deploy to a single tenant, or a handful of tenants in a batch by selecting them manually. This can be an excellent way to get started with multi-tenant deployments, and it's also a good fallback when you need it.
 
 You can also use the Project Overview to deploy to individual tenants by selecting a release and clicking the **Deploy** buttons.
 
-## Deploying to all tenants in an environment
+### Deploying to all tenants in an environment
+
+You can deploy to all of the tenants in an environment using the Octopus UI, the Octopus CLI, or any build-server extensions. Octopus will create a deployment for each tenant, ready for that release to be deployed to the project/environment. Octopus will automatically exclude the tenants who are not ready for that release yet - perhaps they haven't met their lifecycle requirements.
 
-You can deploy to all of the tenants in an environment using the Octopus UI, the Octopus CLI or any of the build-server extensions. Octopus will create a deployment for each tenant which is ready for that release to be deployed to the project/environment. Octopus will automatically exclude the tenants which are not ready for that release yet - perhaps they haven't met their lifecycle requirements.
+As a general rule, we don't recommend deploying to all tenants when there are hundreds of tenants. This may cause an unnecessary overload on your Octopus instance, and this is when we would recommend deploying using [Tenant Tags](/docs/deployment-patterns/multi-tenant-deployments/tenant-tags.md).
```

Comment:
```suggestion
As a general rule, we don't recommend deploying to all tenants when there are hundreds of tenants. This may cause an unnecessary overload on your Octopus instance. Instead, we recommend deploying using [Tenant Tags](/docs/deployment-patterns/multi-tenant-deployments/tenant-tags.md).
```